javascript-loops/main.js

```diff
@@ -0,0 +1,68 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  while (currentNumber < 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+}
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+  while (currentNumber < 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+
+function repeatWord(word, times) {
+  var count = 1;
+  var repeated = '';
+  while (count < times) {
+    repeated += word;
+    count++;
+  }
+  return repeated;
+}
+
+function logEachCharacter(string) {
+  for (var i = 0; i < string.length; i++) {
+    console.log('value of string[i]', string[i]);
+  }
+}
+
+function doubleAll(numbers) {
+  var doubled = [];
+  for (var i = 0; i < numbers.length; i++) {
+    numbers[i] = numbers[i] * 2;
+    doubled.push(numbers[i]);
+  }
+  return doubled;
+}
```

Comment:
`doubleAll` should not modify the contents of the `numbers` array:

---

![image](https://user-images.githubusercontent.com/16143771/152254914-8a6b0816-1ce8-4162-8d71-ce2729cdbd71.png)
****